misc/docs-linter/urls.go

```diff
@@ -53,8 +52,7 @@ func lintURLs(fileUrlMap map[string][]string, ctx context.Context) error {
 		notFoundUrls []string
 	)
 
-	for filePath, urls := range fileUrlMap {
-		filePath := filePath
```

Comment:
This `filePath := filePath` is still necessary here because it is used inside the goroutine.

Reply:
Yup: [21c700d](https://github.com/gnolang/gno/pull/2416/commits/21c700dc4601ca526372ffd772941e019f6eae25)